common/k8stest/util_pvc.go

```diff
@@ -1331,3 +1331,184 @@ func GetPvCapacity(pvName string) (*resource.Quantity, error) {
+			Resources: coreV1.ResourceRequirements{
+				Requests: coreV1.ResourceList{
+					coreV1.ResourceStorage: resource.MustParse(volSizeMbStr),
+				},
+			},
+		},
+	}
+
+	switch volType {
+	case common.VolFileSystem:
+		var fileSystemVolumeMode = coreV1.PersistentVolumeFilesystem
+		createOpts.Spec.VolumeMode = &fileSystemVolumeMode
+	case common.VolRawBlock:
+		var blockVolumeMode = coreV1.PersistentVolumeBlock
+		createOpts.Spec.VolumeMode = &blockVolumeMode
+	}
+
+	// Create the PVC.
+	PVCApi := gTestEnv.KubeInt.CoreV1().PersistentVolumeClaims
+	_, createErr := PVCApi(nameSpace).Create(context.TODO(), createOpts, metaV1.CreateOptions{})
+	if createErr != nil {
+		return "", fmt.Errorf("failed to create pvc: %s, error: %v", volName, createErr)
+	}
+
+	// Confirm the PVC has been created.
+	pvc, getPvcErr := PVCApi(nameSpace).Get(context.TODO(), volName, metaV1.GetOptions{})
+	if getPvcErr != nil {
+		return "", fmt.Errorf("failed to get pvc: %s, namespace: %s, error: %v", volName, nameSpace, getPvcErr)
+	} else if pvc == nil {
+		return "", fmt.Errorf("PVC %s not found, namespace: %s", volName, nameSpace)
+	}
+
+	ScApi := gTestEnv.KubeInt.StorageV1().StorageClasses
+	sc, getScErr := ScApi().Get(context.TODO(), scName, metaV1.GetOptions{})
+	if getScErr != nil {
+		return "", fmt.Errorf("failed to get storageclass: %s, error: %v", scName, getScErr)
+	}
+	if *sc.VolumeBindingMode == storagev1.VolumeBindingWaitForFirstConsumer {
+		return "", nil
+	}
+
+	// Wait for the PVC to be bound.
+	for ix := 0; ix < defTimeoutSecs/timoSleepSecs; ix++ {
+		var pvcPhase coreV1.PersistentVolumeClaimPhase
+		pvcPhase, err = GetPvcStatusPhase(volName, nameSpace)
+		if err == nil && pvcPhase == coreV1.ClaimBound {
+			break
+		}
+		time.Sleep(timoSleepSecs * time.Second)
+	}
+	if err != nil {
+		return "", fmt.Errorf("failed to get pvc status, pvc: %s, namespace:  %s, error: %v", volName, nameSpace, err)
+	}
+
+	// Refresh the PVC contents, so that we can get the PV name.
+	for ix := 0; ix < defTimeoutSecs/timoSleepSecs && pvc.Spec.VolumeName == ""; ix++ {
+		pvc, getPvcErr = PVCApi(nameSpace).Get(context.TODO(), volName, metaV1.GetOptions{})
+		if getPvcErr != nil {
+			return "", fmt.Errorf("failed to get pvc: %s, namespace: %s, error: %v", volName, nameSpace, getPvcErr)
+		} else if pvc == nil {
+			return "", fmt.Errorf("PVC %s not found, namespace: %s", volName, nameSpace)
+		}
+		if pvc.Spec.VolumeName != "" {
+			break
+		}
+		time.Sleep(timoSleepSecs * time.Second)
+	}
+
+	if !local {
+		err = VerifyMayastorPvcIsUsable(pvc)
+		if err != nil {
+			return string(pvc.ObjectMeta.UID), err
+		}
+	}
+
+	logf.Log.Info("Created", "volume", volName, "uuid", pvc.ObjectMeta.UID, "storageClass", scName, "volume type", volType, "size", volSizeMbStr, "elapsed time", time.Since(t0))
+	return string(pvc.ObjectMeta.UID), nil
+}
+
+// RmLocalPVC Delete a PVC in the default namespace and verify that
```

Comment:
Comment say RmLocalPVC  code says RemovePVC

Reply:
updated the RmPVC function to call RemovePVC function https://github.com/openebs/openebs-e2e/pull/23/commits/be7b3f2ee85659903e18b04cf9423c0b9e7cb204